docs/execution-providers/Azure-ExecutionProvider.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Azure
```

Comment:
Suggest to name this "Cloud - Azure" for consistency.

Reply:
Done.